WcaOnRails/config/locales/de.yml

```diff
@@ -3820,6 +4016,9 @@ de:
       probation:
         #original_hash: 1dd6e3b
         name: Delegates auf Bewährung
+      wst_admin:
+        #original_hash: d4112d2
+        name: Software admins
```

Comment:
"Admins" with capital A?

Reply:
Kept a couple of keys as the original (reason: for those keys with less priority than prominent typos, I just wanted to include them to get a nice git diff without deleted / missing keys to make the merge possible)